python/lib/dcoscli/tests/integrations/test_package.py

```diff
@@ -377,10 +377,10 @@ def test_install_missing_package():
 
 
 def test_uninstall_missing():
-    stderr = 'Package [chronos] is not installed\n'
+    stderr = 'Error: Package [chronos] is not installed\n'
     _uninstall_chronos(returncode=1, stderr=stderr)
 
-    stderr = 'Package [chronos] with id [/chronos-1] is not installed\n'
+    stderr = 'Error: Package [chronos] is not installed\n'
```

Comment:
Why did this  error message change?

Reply:
With Go we prefix all errors with `Error:` and because of slightly different logic some additional information may change